src/api/v1/regions/community_businesses.ts

```diff
@@ -0,0 +1,21 @@
+import * as Hapi from 'hapi';
+import { CommunityBusinesses } from '../../../models';
+import { response } from '../schema/response';
+
+
+export default [
+  {
+    method: 'GET',
+    path: '/regions/community-businesses',
```

Comment:
This isn't what I meant. If you look at the message I sent, I used the URL `/regions/:id/community-businesses`, not `/regions/community-businesses`. The semantic being expressed is "for region :id, what community businesses are registered?". 

Is there a reason you want all community businesses/regions in one go?

Reply:
Yes, it is needed for the volunteer app.

Reply:
Is there a PR I can look at? I'm interested to see what it's needed for.

Reply:
`www/js/controllers/register-controller.js` in https://github.com/TwinePlatform/twine-app/pull/21

You can see the UX on the register page of the app.

Reply:
Originally it worked fetching for a single region, like I imagined. Now we're grabbing all the data in one go.

I don't think it's more performant this way tbh. If you imagine that in the most likely use case, a volunteer will know what region they're in, so fetching by region will result in less data transferred.